src/components/header/index.tsx

```diff
@@ -1,17 +1,26 @@
 import { h } from 'preact'
 import { header, back as backClass } from './style.sss'
 import Icon from '../icon'
+import { route } from 'preact-router'
 
 interface HeaderProps {
   title?: string
   back?: string
   contents?: any
+  verify?: boolean
 }
 
-const Header = ({ title, back, contents }: HeaderProps) => (
+const Header = ({ title, back, contents, verify }: HeaderProps) => (
   <header class={header}>
     {back && (
-      <a class={backClass} href={back}>
+      <a
+        class={backClass}
+        onClick={() =>
+          !verify || confirm('Are you sure you want to leave?')
```

Comment:
Would this be clearer as:

```js
verify && .......
```

Reply:
I think: "If we don't need to verify, or we got a successful verification" is as clear as "If we need to verify and we got a successful verification".